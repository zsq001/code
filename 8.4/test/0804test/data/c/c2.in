1000
666531 902375 358897 250978 459562 65107 472436 372417 715558 622539 546922 765348 351108 344575 115582 767576 424945 534909 687221 870014 990554 508988 830094 988240 557120 128357 19071 389498 177121 665332 941936 765773 946403 824405 989481 321331 963670 286294 411990 993205 793351 533145 477714 697285 342350 130928 531181 613824 846727 230635 489686 161802 174875 740095 345381 207841 118673 407231 206389 391728 606926 1145 809394 850416 197048 799197 588191 853236 323416 632630 851951 351494 557530 865426 696274 287285 471131 690713 917177 730516 964649 86386 950995 786674 622341 596819 774831 92241 72986 714989 682882 548074 58071 558940 920067 593713 927999 169779 44627 627911 27796 30637 228414 72063 992487 988590 720154 863072 293356 875675 89065 81564 165990 294799 69808 73114 918379 328691 831742 390726 46485 114045 45352 968575 689955 93406 691932 412778 999202 564508 185125 572166 848935 742744 894319 672017 679884 946354 965924 941903 4349 664457 475893 634522 873869 123699 543317 504585 946562 50290 709388 627327 442369 883154 690621 910660 313908 460878 761606 855507 30038 499005 379216 200260 414079 993536 596188 597862 498431 320538 860572 331610 780903 508144 712529 593490 681391 377065 268083 677664 626794 247673 650140 11332 253888 736910 837970 779245 534339 299247 531096 563266 216931 857542 484066 210748 842955 431142 771586 489481 974242 743174 843359 213693 158029 365592 974359 230003 574560 551877 961916 584497 264592 539043 619125 515826 711353 426159 49663 532024 771683 919464 219039 487687 872352 979552 148152 791141 841788 961315 896234 354579 827059 970640 350476 745468 483865 204493 699432 835568 964159 508564 482503 741414 606675 481594 507938 435329 687402 200686 17817 956681 347248 711268 716913 649132 639830 153102 253559 443209 781460 328316 105064 38080 134070 773355 403441 550435 527130 56454 830724 722683 503655 919384 400539 10403 3963 997991 230222 838884 913427 734100 46918 629557 864213 415279 374666 144119 778821 364659 578584 651594 890390 902859 455363 557193 471941 228071 532060 56050 65068 378485 782846 193400 103968 36028 157413 906735 277094 271916 119258 762571 640688 200243 306276 314335 332104 991837 818102 146357 387520 817171 743052 915655 507720 880489 624168 702849 45713 965176 790115 888207 174007 447090 375573 919824 378884 530148 967927 18301 355333 73549 314642 825841 673890 350857 503742 761764 218894 751104 15578 51931 677729 270398 152286 72808 410716 765012 683521 610551 724812 304184 321642 10665 645305 521521 574946 417332 276128 646046 54378 324453 948181 534784 139793 702802 735153 952628 538505 320989 907160 705199 932585 614329 776276 96760 367485 469224 904225 36318 832717 113759 5086 536329 709904 151366 741350 135384 330694 264381 248657 216064 660154 680740 758209 616722 759954 185944 44099 175926 48679 198590 990094 890702 625093 122730 643774 819229 853559 877174 325424 489052 583256 270449 106659 759801 896301 394748 618171 406832 798205 155524 685450 712428 104169 738401 781682 200324 593281 756079 103267 721226 553189 138896 906672 448995 855425 279713 527778 269126 347965 399226 716301 276358 155998 28223 13367 122875 492210 550007 173343 876608 916443 116307 812668 647139 657443 688302 536181 216730 597749 272 414351 377824 304586 970210 675493 557303 117191 818746 102631 884780 61427 266092 664990 864295 216982 108900 152144 261690 355687 56497 205024 45782 218575 473868 824643 501076 271215 298079 160437 794865 243466 296682 324213 601193 197812 860880 102698 564876 207804 743652 317220 176599 32794 2813 703320 903900 801230 849145 170803 400159 161119 133101 86600 29724 685024 159366 84860 648609 35660 184533 462722 658003 813397 933486 387642 585056 29172 947862 63596 420610 783022 950840 913768 444588 431825 450732 790573 794659 839667 102831 752746 959096 974671 452315 488411 959352 550929 221016 216988 151891 901431 161119 143453 885162 890319 766832 438712 288465 145115 21912 403247 881191 213840 934147 419727 349364 350926 436625 988967 728739 659537 782147 572826 464799 362436 970908 241492 27079 592981 964526 642372 892138 178069 659976 320494 148070 438158 590659 419358 192306 844725 150721 83547 673097 90873 846546 888539 613858 547155 20143 497567 913623 294587 991956 713616 952132 142062 80824 561598 409315 340788 393092 240299 186204 217444 506604 998078 891339 177089 165593 498201 769878 800134 425591 92541 52944 957103 70943 523775 357770 597465 623152 368792 985133 143462 778963 572745 744777 927326 992972 275507 432086 596270 839637 765692 945067 681925 952878 781441 148093 137085 85543 277038 278047 706394 787026 420172 313378 604214 98116 839170 981927 851576 637273 674834 98818 724628 813540 956440 149594 847486 303534 737065 538788 289110 345327 638164 821173 928286 950336 814878 511090 619001 75572 65353 652818 636487 31518 214822 778574 654048 135060 282462 866266 626722 7541 42339 589397 644334 573780 599330 744292 854678 182630 360094 340626 281728 57069 196544 834420 711011 340584 491154 159796 131002 224307 611365 615745 129509 894783 247360 673708 793260 546927 764859 845990 581758 138240 756707 894149 643531 496252 615947 276756 451855 50010 711881 84598 736427 632580 603184 319216 434879 219378 689956 989465 358916 899031 940196 530385 729733 206156 167482 588988 621453 968178 36205 9392 918904 633475 291071 235692 372595 457018 288799 637761 850720 243415 125855 484810 607645 659051 971435 994007 279630 824960 478131 585262 934130 40338 953637 96539 462819 489373 257236 398565 773353 310204 375612 532927 479774 40342 738153 143765 733585 14301 559577 507438 21067 75496 188221 962780 693876 263420 343408 248001 440546 386721 910256 696440 840384 630961 887480 279440 482913 845833 432655 858961 441396 877488 68860 925388 202453 83367 421920 904517 400744 710695 947572 318082 651860 703455 73440 630858 450271 817120 628768 586863 323703 105038 760595 566096 690275 937935 898816 956839 684571 837903 729353 427340 328347 978116 294318 436960 783531 139942 937241 338572 940279 859820 634080 540297 832425 288625 221548 581270 487833 128755 84030 359612 871841 614310 7459 725368 476875 920128 286435 970134 387321 864873 206176 249126 504319 943190 838162 679803 227989 604910 3500 587349 669015 877625 902723 143389 23877 18302 230306 412214 122681 234877 415928 509184 110242 231999 187407 831964 140677 651325 988590 32917 179622 549621 234733 3907 347897 258167 521587 574747 80084 772006 711422 762116 525794 828237 567967 421381 633851 253395 360321 285462 975205 673695 905821 449019 154275 71129 599866 525503 493258 315498 148017 635710 331971 540461 950678 477403 179326 683080 280843 817268 499987 251554 465424 722971 808507 221481 566007 210019 90524 383859 816818 701680 435548 361710 309187 727069 124120 181568 191833 534934 841333 65321 612158 595006 270001 252230 926001 143467 461372 759400 23547 956062 514320 370494 93144 602971 123201 265132
4
5
2
6
10
