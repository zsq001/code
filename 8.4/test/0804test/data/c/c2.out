54
203
42
17
